pkg/test/testobjects.go

```diff
@@ -0,0 +1,21 @@
+package test
```

Comment:
Unnecessary package, these are only used in the cdc package.

Reply:
I can move this into the CDC package if you prefer. My logic in separating it is that hopefully it will be re-used elsewhere. I think all of our projects should have a `pkg/test` package which holds functions that can produce minimal examples of the top level (CR-linked) objects they produce.

Reply:
OK, will do.